src/app/airdrop/section-6.tsx

```diff
@@ -19,24 +19,14 @@ export const Section6 = () => {
       .then((response) => response.json())
       .then((data) => {
         const users: LeaderboardUser[] = data.leaderboard.map(
-          (leaderboardUser: LeaderboardData) => {
-            const lUser = data.users.find(
-              (u: User) => u.id === leaderboardUser.user_id,
-            );
-            const stats = data.userStats.find(
-              (s: UserStats) => s.user_id === leaderboardUser.user_id,
-            );
+          (leaderboardUser: LeaderboardData, index: number) => {
             return {
-              id: leaderboardUser.user_id,
-              name: lUser.username,
-              wallet: lUser.wallet_address,
-              boss_score: stats.boss_score,
-              builder_score: stats.builder_score,
-              nominations: stats.nominations,
-              rank: leaderboardUser.rank,
-              highlight: lUser.wallet_address === user?.wallet_address,
+              id: index,
```

Comment:
This `id` here will end up being used as a `key` for a React List. 
Using `index` as an ID is a bad practice since it defeats the purpose of the key. Not very impactful in this particular case, but should be avoided. 

[Reference](https://react.dev/learn/rendering-lists#why-does-react-need-keys)

Reply:
I was not aware. Thank you.